apis/v1alpha1/httproute_types.go

```diff
@@ -78,6 +88,23 @@ type HTTPRouteHost struct {
 	ExtensionRef *RouteHostExtensionObjectReference `json:"extensionRef"`
 }
 
+// RouteTLSConfig describes a TLS configuration defined at the Route level.
+type RouteTLSConfig struct {
+	// CertificateRef is the reference to Kubernetes object that
+	// contain a TLS certificate and private key.
+	// This certificate MUST be used for TLS handshakes for the domain
+	// this RouteTLSConfig is associated with.
+	// If an entry in this list omits or specifies the empty
+	// string for both the group and the resource, the resource defaults to "secrets".
```

Comment:
This could use some clarification. The default group is "core" and the default resource is "secrets". The current wording makes it seem like both need to be left empty for the default to apply.

Reply:
I copied this over from https://github.com/kubernetes-sigs/service-apis/blob/master/apis/v1alpha1/tlsconfig_types.go#L59-L60. Can we change this outside this PR?